src/components/counter/counter.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./counter.css";
+
+class Counter extends React.Component{
+    state = {
+        counterValue : 0
+    };
+
+incrementCounter(){
+    let initialValue = this.state.counterValue;
```

Comment:
I would advise to use the second form of setState:
`this.setState((state) => ({
  counteValuer: state.counterValue + 1,
}));`

Reply:
thanks changed